testdata/vast4_universal_ad_id.xml

```diff
@@ -43,7 +43,7 @@
             </MediaFiles>
             <VideoClicks>
               <ClickThrough id="blog">
-                <![CDATA[https://iabtechlab.com]]>
+                <![CDATA[https://test.com]]>
```

Comment:
it's still .com tld, please use .test tld

Reply:
sure!